iothub/device/src/Common/CommonConstants.cs

```diff
@@ -75,6 +75,9 @@ static class CommonConstants
         public const string AdminUriFormat = "/$admin/{0}?{1}";
         public const string DefaultConfigurationKey = "_default_config_key";
 
+        // Security message constants
+        public const string SecurityMessageInterfaceId = "urn:azureiot:Security:SecurityAgent:1";
+
```

Comment:
Is this to align with the concept of "interface?"

Reply:
Yes, the ASC team got this from the PnP guys after weeks of deliberation.

---
In reply to: [300181938](https://github.com/Azure/azure-iot-sdk-csharp/pull/999#discussion_r300181938) [](ancestors = 300181938)